packages/core/src/external/athenahealth/get-patient.ts

```diff
@@ -50,8 +54,17 @@ export async function getPatient({
     return patientResourceSchema.parse(resp.data);
     // eslint-disable-next-line @typescript-eslint/no-explicit-any
   } catch (error: any) {
-    console.log(error);
     if (error.response.status === 404) return undefined;
+    const msg = `Failure while getting patient @ AthenHealth`;
+    log(`${msg}. Patient ID: ${patientId}. Cause: ${errorToString(error)}`);
+    capture.error(msg, {
+      extra: {
+        baseUrl,
+        patientId,
+        context: "athenahealth.get-patient",
+        error,
+      },
```

Comment:
They don't include a reference number/id of sorts, like CW does, do they?

Reply:
I don't think so unfortunately.